mvp-spec/publishers-requirements.md

```diff
@@ -575,9 +504,9 @@ an example:
 ```
 <!--partial-end-->
 
-### Step 7: Display the Addressable Content and make the Audit Log available
+### Step 7: Display the ad and make the Audit Log available
```

Comment:
Why not Addressable Content?
Concretely, Addressable Content == Ad. right?
If possible, I would like that reduce the glossary to the minimum to avoid to the complexity of the documentation.

Maybe to be discussed with Joshua to see if we can replace the Addressable Content by Ad everywhere. Else, I would prefer to keep Addressable Content, here.

Reply:
Yes, I plan on removing "Addressable Content" everywhere, as it's a very cumbersome way of saying "ad" (and I've already removed "Addressable Content" from the glossary.